packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -35,6 +35,12 @@ import 'thumb_painter.dart';
 /// that use a slider will listen for the [onChanged] callback and rebuild the
 /// slider with a new [value] to update the visual appearance of the slider.
 ///
+/// {@tool dartpad}
+/// This example shows [CupertinoSlider] in action.
```

Comment:
This description doesn't really give much information: the reader already knows that it's an example that should what the slider does.  Maybe something more about what it is teaching, like: "This example shows how to show the current slider value as it changes."

Reply:
That makes sense